lib/rdoc/code_object/class_module.rb

```diff
@@ -831,22 +827,6 @@ def update_includes
     includes.uniq!
   end
 
-  ##
-  # Deletes from #extends those whose module has been removed from the
-  # documentation.
-  #--
-  # FIXME: like update_includes, extends are not reliably removed
-
-  def update_extends
```

Comment:
I think we should keep this method. It should be called from method `complete` just like `update_includes`

Example: "Extended With Modules" section of https://docs.ruby-lang.org/en/master/CGI.html
```
Extended With Modules
CGI::Util
CGI::Util
```
Note that other modules (`QueryExtension`, `Html3`, etc) are just wrongly listed by `rdoc/parser/ruby.rb`

Reply:
https://github.com/ruby/rdoc/pull/1317 for fixing the issue